### PR TITLE
Added placeholder for Amalgamating Business application

### PR DESCRIPTION
**Pull request by jamespaologarcia**
*Issue #:* /bcgov/entity#19930

*Description of changes:*

Added a condition to have a placeholder for empty 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/resources/v2/business/business.py [hunk 1]
```diff
@@ -189,6 +189,8 @@ def search_businesses():
                 draft['legalName'] = (Business.BUSINESSES
                                       .get(draft_dao.json_legal_type, {})
                                       .get('numberedDescription'))
+                if draft['legalName'] == {}:
```
Comment (vysakh-menon-aot) [inline legal-api/src/legal_api/resources/v2/business/business.py:192]:
```suggestion
                if draft['draftType'] == 'ATMP' and not draft['legalName']:
```

File: legal-api/tests/unit/resources/v2/test_business.py [hunk 2]
```diff
@@ -338,6 +338,48 @@ def test_get_business_with_court_orders(session, client, jwt):
     assert rv.json['business']['identifier'] == identifier
     assert rv.json['business']['hasCourtOrders'] == True
 
+@pytest.mark.parametrize('identifier, legal_type, draft_type, nr_number', [
+    ('Tb31yQIuBw', 'COMP', 'ATMP', None),
```
Comment (vysakh-menon-aot) [inline legal-api/tests/unit/resources/v2/test_business.py:342]: there is no legal_type called COMP. Did you mean BC (https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/business.py#L66)?

File: legal-api/tests/unit/resources/v2/test_business.py [hunk 3]
```diff
@@ -338,6 +338,48 @@ def test_get_business_with_court_orders(session, client, jwt):
...
+    # Setup a temporary registration and draft filing
+    temp_reg = RegistrationBootstrap()
+    temp_reg._identifier = identifier
+    temp_reg.save()
+
+    json_data = copy.deepcopy(FILING_HEADER)
+    json_data['filing']['header'] = {
+        'name': 'amalgamationApplication',
+        'identifier': identifier,
+        'legalType': legal_type,
```
Comment (vysakh-menon-aot) [inline legal-api/tests/unit/resources/v2/test_business.py:356]: these are not header props

File: legal-api/tests/unit/resources/v2/test_business.py [hunk 4]
```diff
@@ -338,6 +338,48 @@ def test_get_business_with_court_orders(session, client, jwt):
...
+    }
+    json_data['filing']['amalgamationApplication'] = {
+        'type': 'regular',
+        'nameRequest': {
+            'nrNumber': nr_number,
```
Comment (vysakh-menon-aot) [inline legal-api/tests/unit/resources/v2/test_business.py:361]: legalType is mandatory in nameRequest object (https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/name_request.json#L53)

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Pull request merged**